### PR TITLE
[Docs] Use system preference for docs theme

### PR DESCRIPTION
**Pull request by dominikandreasseitz**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -82,29 +82,25 @@ theme:
   logo: assets/logo/qadence_logo_small_white.svg
   features:
   - content.code.annotate
-  - content.action.view
-  - content.action.edit
```
Comment (jpmoutinho) [inline mkdocs.yml:86]: This was cluttering the space around the logo... it's never really used

File: README.md [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
+
 <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="./docs/assets/logo/qadence_logo_white.svg">
-  <source media="(prefers-color-scheme: light)" srcset="./docs/assets/logo/qadence_logo.svg">
-  <img alt="Qadence logo" src="./docs/assets/logo/qadence_logo.svg">
+  <source media="(prefers-color-scheme: dark)" srcset="./assets/logo/qadence_logo_white.svg">
+  <source media="(prefers-color-scheme: light)" srcset="./assets/logo/qadence_logo.svg">
+  <img alt="Qadence logo" src="./assets/logo/qadence_logo.svg">
 </picture>
+</div>
```
Comment (jpmoutinho) [inline README.md:8]: Ok apparently this broke the github readme

Comment (jpmoutinho) [reply README.md:8]: Ah wait

**Review by jpmoutinho — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (dominikandreasseitz) [reply README.md]: ?

**Comment by jpmoutinho** [comment]
Ok @dominikandreasseitz I checked this again and hopefully it will work now.

Unfortunately getting mkdocs material and markdown shenanigans to all work together seems to be a nightmare. I couldn't find a setting where logo resizing + logo changing to light and dark + the docs changing according to the system default would all work together.

Currently it does not resize the logo (I can later take care of that manually), but should be changing it correctly with the toggle button. Furthermore when the user first loads the website it should pre-define to light or dark according to the browser settings.

**Pull request merged**